### ISSUE TITLE
Import Variables File, Variable name shows lowercase

### ISSUE THREAD

**Issue by XiaoyaoRay** [nokia/RED#243]
Variable file:
   ```
    .......
    MYSQL_HOST_IP = "xxxxx"    
    MYSQL_ROOT_PWD = "xxxx"  
    MYSQL_ROOT_PORT = xxx
    ......
  ```
 RED:
![image](https://user-images.githubusercontent.com/25135479/45139000-f53ce700-b1e0-11e8-996f-75cce41afd15.png)

Variable name can shows uppercase ?

**Comment by michalanglart** [comment]
Hi,

RED uses some Robot code in order to load variables from files. The data structure which holds the variables from file have normalized names so that it is possible for Robot to be case-insensitive.

It should be however rather easy to fetch the names as they are in file instead of their normalized variants.

**Comment by michalanglart** [comment]
Closing - released in 0.8.8
